templates/enter2fa.html

```diff
@@ -20,6 +20,7 @@ <h2>Confirm you're you!</h2>
                             <i class="material-icons prefix">keyboard</i>
                             <input class='validate otp' type='number' id='otc' max="999999" minlength="6" maxlength="6"/>
                             <span class="helper-text" data-error="Must be 6 digits" data-success="right"></span>
+							<button type="button" class="waves-effect waves-light btn" id="otc-button">Submit</button>
```

Comment:
Was there an issue that made you add a submit button here? Normally it should auto-submit once the 6 digits have been entered.

Reply:
Yes, auto submit on keypress does not work in a mobile browser.